openshift/prometheus-route.yml

```diff
@@ -0,0 +1,13 @@
+apiVersion: route.openshift.io/v1
```

Comment:
We don't expose Prometheus in Kubernetes, should we for OpenShift?

Reply:
I can remove that, in OpenShift there is not another way to access the Prometheus UI without a route the way it is deployed. Under most operations I would expect A user not needing to access it.